src/main/java/seedu/address/logic/commands/UnlabelCommand.java

```diff
@@ -0,0 +1,115 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_LABEL_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG_NAME;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.label.Label;
+import seedu.address.model.tag.FileAddress;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.TagName;
+import seedu.address.model.tag.TagNameEqualsKeywordPredicate;
+
+/**
+ * Deletes a specified tag's label(s).
+ */
+public class UnlabelCommand extends Command {
+    public static final String COMMAND_WORD = "unlabel";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Deletes Label(s) in an existing Tag. "
+            + "\nParameters: "
+            + PREFIX_TAG_NAME + "TAG_NAME "
+            + PREFIX_LABEL_NAME + "LABEL_NAME "
+            + "[" + PREFIX_LABEL_NAME + "LABEL_NAME" + "]...\n"
+            + "\nExample: " + COMMAND_WORD + " "
+            + PREFIX_TAG_NAME + "cs2103 "
+            + PREFIX_LABEL_NAME + "notes "
+            + PREFIX_LABEL_NAME + "CS2103T";
+
+    public static final String MESSAGE_SUCCESS = "Tag successfully updated: %1$s";
+    public static final String MESSAGE_LABEL_MISSING = "All of this label can't be found in %s:\n";
```

Comment:
Is it better to say "All of these labels can't be found"?

Reply:
At first, I am writing it to be similar like that, but I thought it is better if we can show which tag we are talking about in the CommandResult.